vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java

```diff
@@ -77,7 +77,7 @@ private ResourceBundle extractUnsafeBundle(Object bundle, Locale locale) {
 			try {
 				return ResourceBundle.getBundle(baseName, locale);
 			} catch (MissingResourceException e) {
-				logger.warn("couldn't find message bundle, creating an empty one", e);
+				logger.warn("couldn't find message bundle, creating an empty one");
```

Comment:
could you also add the `baseName` and `locale` on the message? It'd look like this:

>  couldn't find message bundle for base name messages, locale en_US. creating ...